homeassistant/components/verisure/camera.py

```diff
@@ -21,29 +22,37 @@ def setup_platform(
     discovery_info: dict[str, Any] | None = None,
 ) -> None | Literal[False]:
     """Set up the Verisure Camera."""
-    if not int(hub.config.get(CONF_SMARTCAM, 1)):
+    coordinator = hass.data[DOMAIN]
+    if not int(coordinator.config.get(CONF_SMARTCAM, 1)):
         return False
```

Comment:
Nothing is checking this return value.
```suggestion
        return
```